presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -102,6 +103,7 @@
     private int re2JDfaStatesLimit = Integer.MAX_VALUE;
     private int re2JDfaRetries = 5;
     private RegexLibrary regexLibrary = JONI;
+    private StatisticalDigestImplementation statisticalDigestImplementation = StatisticalDigestImplementation.TDIGEST;
```

Comment:
There's a change in behavior that this entails.  We definitely don't want this to be the default value.